packages/common/src/adapters/notification.ts

```diff
@@ -0,0 +1,500 @@
+import { full } from '@audius/sdk'
+
+import type { BadgeTier, ID } from '~/models'
+import type { ChallengeRewardID } from '~/models/AudioRewards'
+import type { StringUSDC, StringWei } from '~/models/Wallet'
+import {
+  Achievement,
+  Entity,
+  NotificationType,
+  type Notification
+} from '~/store/notifications/types'
+import { decodeHashId } from '~/utils/hashIds'
+import { removeNullable } from '~/utils/typeUtils'
+
+function formatBaseNotification(notification: full.Notification) {
+  const timestamp = notification.actions[0].timestamp
+  return {
+    groupId: notification.groupId,
+    timestamp,
+    isViewed: !!notification.seenAt,
```

Comment:
So originally notifs were on _identity_, which never used snake casing! We had just retrofitted DNs response to match identity rather than updating the components

Reply:
Oh joy 😆 

Looking forward to the day where we use hashIds and camelCase directly and these adapters aren't even necessary.